.github/workflows/run_all_tests.yml

```
@@ -162,7 +158,7 @@ jobs:
         run: |
           conda config --set always_yes yes --set changeps1 no
           conda info
-          # the conda dependency resolution for tox under python 3.7 can install the wrong importlib_metadata
+          # the conda dependency resolution for tox under python 3.8 can install the wrong importlib_metadata
           conda install -c conda-forge tox "importlib_metadata>4"
```

```suggestion
          conda install -c conda-forge tox
```
The importlib_metadata install from conda-forge was necessary under python 3.7 but is no longer needed if we drop testing on python 3.7